source/server-selection/server-selection.rst

```diff
@@ -370,14 +374,15 @@ Components of a read preference
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 A read preference consists of a ``mode`` and optional
-``tag_sets`` and ``maxStalenessSeconds``.  The ``mode`` prioritizes between primaries and
-secondaries to produce either a single suitable server or a list of candidate
-servers.  If ``tag_sets`` and ``maxStalenessSeconds`` are set, they determine
-which candidate servers are eligible for selection.
+``tag_sets``, ``maxStalenessSeconds``, and ``hedge``.  The ``mode`` prioritizes
+between primaries and secondaries to produce either a single suitable server or
+a list of candidate servers.  If ``tag_sets`` and ``maxStalenessSeconds`` are
+set, they determine which candidate servers are eligible for selection. If
+``hedge`` is set, it configures how server hedged reads are used.
 
 The default ``mode`` is 'primary'.  The default ``tag_sets``
 is a list with an empty tag set: ``[{}]``. The default ``maxStalenessSeconds``
-is -1 or null, depending on the language.
+is -1 or null, depending on the language. The default ``hedge`` is empty.
```

Comment:
Relates to my comment below, but should we differentiate between "empty" and "unset"? If the user provides an empty BSON document for `hedge`, should we send that to the server?

It may be more intuitive for users if we treat unset and empty the same; however, that would leave no way for an application to indicate that it wants to use `hedge` and fully rely on the server-side defaults for its options (`enabled` and `delay`). Perhaps this is something to ask one of the server engineers or product managers to determine what the desired user-facing behavior is.

Reply:
I changed "empty" to "null" in the sentence above, but we may also choose a less language-specific "unset".

Submitting an empty `hedge` document would be the preferred way to turn on hedging with server defaults, although we may treat this as unset in favour of using `{ hedge: { enabled: true } }` to be more explicit.

Reply:
I would prefer "unset" over "null", as there's no chance it could be misinterpreted a present field with the BSON null value.